test/screenshot/top-app-bar/NavIcon.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const NavIcon = () => {
```

Comment:
What if, we just added the class to the NavIcon within the TopAppBar Component?

Reply:
Also we will need to do this to actionItems as well. Let me know how you would like to proceed. Instead of having these icons we could pass it on to the end developer, but just add the class `mdc-top-app-bar__action-item`.

Both actionItemButton, actionItemAnchor, and navIcon are easy, but I think we want them to be npm components as well.

Reply:
talked offline, Matt will intercept the elements and add the `mdc-top-app-bar__navigation-icon` and `mdc-top-app-bar__action-icon` to the elements. And update the test NavIcon.js and actionItems.js to actually be React.Component.